webpack.config.js

```diff
@@ -14,11 +13,11 @@ const {
 const ENV = process.env.NODE_ENV;
 const PATHS = {
   MODULES: 'node_modules',
+  THIRDPARTY: 'thirdparty',
```

Comment:
I recommend using `Path.resolve()` so that it uses absolute paths to resolve this folder